LICENSE

```diff
@@ -0,0 +1,14 @@
+Copyright 2015 Workiva Inc.
+Portions Copyright 2016 The Chromium Authors. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
That's what you do to each source file. You still need to include the actual license (see section 4a).